charts/apisix/values.yaml

```diff
@@ -186,6 +183,11 @@ admin:
     ipList:
       - 127.0.0.1/24
 
+nginx:
+  workerRlimitNofile: "1047552"
+  workerConnections: "16384"
```

Comment:
we should keep the default value, right?

Reply:
resolved